common.gypi

```diff
@@ -366,6 +366,9 @@
             'cflags': [ '-I/usr/local/include' ],
             'ldflags': [ '-Wl,-z,wxneeded' ],
           }],
+          ['OS=="linux"', {
+            'ldflags': [ '-lrt' ],
+          }]
```

Comment:
this reminds me of something. wasn't it recently removed?

Reply:
unfortunately it s needed in some old systems for clock_gettime api

Reply:
Our baseline is glibc 2.17 and that version moved librt into libc. See 7ec558cb243c7ffc3a120e29dc43ed9045a8b0a0 for why this change breaks the build on some supported systems.